Cargo.toml

```diff
@@ -56,13 +51,15 @@ memchr = "2.3"
 uuid = { version = "1.0", optional = true }
 smallvec = "1.6.1"
 cast = { version = "0.3", features = ["std"] }
-arrow = { version = "40", default-features = false, features = ["prettyprint", "ffi"] }
+arrow = { version = "39.0.0", default-features = false, features = ["prettyprint", "ffi"] }
 rust_decimal = "1.14"
 strum = { version = "0.24", features = ["derive"] }
 r2d2 = { version = "0.8.9", optional = true }
 calamine = { version = "0.19.1", optional = true }
 num = { version = "0.4", optional = true, default-features = false, features = ["std"] }
 duckdb-loadable-macros = { version = "0.1.0", path="./duckdb-loadable-macros", optional = true }
+polars = { version = "0.30.0", features = ["dtype-full"], optional = true}
+arrow2 = { version = "0.17.2", features = ["arrow"], optional = true }
```

Comment:
if we can use the re-export arrow2 in polars, then we can remove this dependency entry?

Reply:
https://github.com/pola-rs/polars/blob/4ce40409d1c2a4bbb70289027137262b78720eaa/Cargo.toml#L43

Currently we cannot, because `polars` does not turn on feature `arrow` for `arrow2`.